### PR TITLE
Do not build tools if --disable-tools is set

### PR DESCRIPTION
**Pull request by ffontaine**
Signed-off-by: Fabrice Fontaine <fabrice.fontaine@orange.com>

### CODE REVIEW

File: tools/Makefile.am [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 include $(abs_top_nlbuild_autotools_dir)/automake/pre.am
 
+if OPENTHREAD_BUILD_TOOLS
```
Comment (jwhui) [inline tools/Makefile.am]: I think we still want to include these directories in `DIST_SUBDIRS` even when `--disable-tools` is set.  Instead, we should wrap the automake rules that involve building in each tool-specific `Makefile.am`.

**Comment by jwhui** [comment]
@ffontaine, this PR looks good.  You'll need to sign the CLA before we can merge this.  Thanks.

**Comment by jwhui** [comment]
@ffontaine, will you be able to sign the CLA so that we can merge this?

**Comment by ffontaine** [comment]
I signed it!

**Pull request merged**